### DISCUSSION TITLE
i have flutter app and then i start build ios app only by selecting ios checkbox only but i got error "Skip publishing to Google Play: no valid release APKs found" in publishing phase

### DISCUSSION THREAD

**Discussion by bhaumik1111** [codemagic-ci-cd/codemagic-docs#2185] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i start building ios only from my flutter app but i got the error in publishing phase

"Skip publishing to Google Play: no valid release APKs found"

### Expected results

it has to be go to publishing process 

### Actual results

== Gathering artifacts ==

== Publishing artifacts ==

Publishing artifact Today.ipa
Publishing artifact Runner.app.zip
Publishing artifact Runner.app.dSYM.zip
Skip publishing to Google Play: no valid release APKs found

**Comment by dtrdic** [comment]
@bhaumik1111 please share a build id so i can take a closer look

**Reply by bschmalb** [reply]
@dtrdic I have the same problem here but with a different use case. 
I'm building a iOS and android version to publish to firebase app distribution but **NOT** to Google Play or to the App Store.
Build-id: 65535277b7413cd992628c10

**Reply by dtrdic** [reply, marked as answer]
hey @bschmalb i can see this error in the logs "_This project is not linked to a Google Play account_".
Firebase cannot find a connection between your Firebase project and a Google Play account. This linkage is essential for App Distribution using Firebase. Please check out [this official Firebase guide](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before-you-begin) on how to create a Firebase link to Google Play

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion due to a lack of activity. You can open the discussion again by replying to this message or opening a new discussion.
